legislators-social-media.yaml

```diff
@@ -3478,6 +3480,7 @@
     thomas: '01654'
     govtrack: 400074
   social:
+    twitter: lacyclaymo1
```

Comment:
Have you guys read the content of both the Twitter feed and the website? They both go back years. Excluding it on a technicality seems a bit silly.